macros/NEXT100_Ar_alpha_full.config.mac

```diff
@@ -0,0 +1,56 @@
+## ----------------------------------------------------------------------------
+## nexus | NEXT100_Ar_alpha_full.config.mac
+##
+## Initialization macro to alpha particle decays in the active
+## volume of the NEXT-100 detector during the low pressure run.
+##
+## The NEXT Collaboration
+## ----------------------------------------------------------------------------
+
+##### VERBOSITY #####
+/run/verbose 0
+/event/verbose 0
+/tracking/verbose 0
+
+/process/em/verbose 0
+
+##### GEOMETRY #####
+/Geometry/Next100/elfield true
+/Geometry/Next100/EL_field 6.7 kV/cm
+/Geometry/Next100/max_step_size 1. mm
+/Geometry/Next100/pressure 4.2 bar
+/Geometry/Next100/gas GAr
+/Geometry/PmtR11410/time_binning 25. nanosecond
+/Geometry/Next100/sipm_time_binning  1. microsecond
+
+## Geant4 only permits units of mm/ns via the macro
```

Comment:
You can also define a new unit, if you prefer.

Reply:
Didnt know thats how it worked, updated now